packages/lib-runtime/src/store/PluginLoader.ts

```diff
@@ -202,8 +202,7 @@ export class PluginLoader {
    * Register the global function used by plugin entry scripts.
    */
   registerPluginEntryCallback(getWindow: () => typeof window = _.constant(window)) {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const windowGlobal = getWindow() as any;
+    const windowGlobal = getWindow() as unknown as AnyObject;
```

Comment:
This change was done to reduce our existing `@typescript-eslint/no-explicit-any` rule suppressions.